tests/compile-fail/for_loop.rs

```diff
@@ -286,39 +286,86 @@ fn main() {
         id_col[i] = 1f64;
     }
 
-    /*
+    // This is fine.
     for i in (10..0).map(|x| x * 2) {
         println!("{}", i);
-    }*/
+    }
 
-    for _v in vec.iter() { } //~ERROR it is more idiomatic to loop over `&vec`
-    for _v in vec.iter_mut() { } //~ERROR it is more idiomatic to loop over `&mut vec`
+    for _v in vec.iter() { }
+    //~^ ERROR it is more idiomatic to loop over `&vec`
+    //~| HELP to write this more concisely, try looping over
+    //~| SUGGESTION &vec
```

Comment:
I usually prefer to had a bit of context around the suggestion, just to be sure:

```rust
//~| SUGGESTION for _v in &vec {
```